peps/pep-0661.rst

```diff
@@ -197,6 +191,50 @@ add minimal complexity.
 
 Ordering comparisons are undefined for sentinel objects.
 
+Typing
+------
+
+To make usage of sentinels convenient in typed Python code, we propose to
```

Comment:
In my view this is about much more than convenience: Not having clear type annotations is one of the 3 main deficiencies of existing implementations of sentinels, as outlined in the "Motivation" sections.

The authors of this PR have much more experience than I regarding typing in Python, but I think at this point we can agree that typing considerations are more than an "afterthought" for convenience.

As a simple change, perhaps we could use "clear and simple" rather than "convenient" here?

Reply:
```suggestion
To make usage of sentinels clear and simple in typed Python code, we propose to
```
Sure, that works too.